src/lib/controller/message_handler.ts

```diff
@@ -192,6 +192,9 @@ export class ControllerMessageHandler {
       case ControllerCommand.supportsFeature:
         const supported = driver.controller.supportsFeature(message.feature);
         return { supported };
+      case ControllerCommand.inclusionState:
+        const inclusionState = driver.controller.inclusionState;
```

Comment:
Would it make sense to add a command to just dump the whole controller state ?

Reply:
Or even better, are events emitted so we can automatically keep this up to date after we receive the initial state in Python?

Reply:
Re: your first question - Yes I'm planning to add a command like that to the controller and node. 

Re: your second question: https://github.com/zwave-js/zwave-js-server/pull/465#issuecomment-994967541

Reply:
If we add a command to dump the whole state, do we need a command for just this one ?

Reply:
- [ ] I suppose we don't. We need to pick a pattern for getting and setting attributes (there are three settable attributes on a Node, two of which have an additional complexity because they can be set locally on the driver and on the node IF it has a given command class. I'm ok with using a state dump as a way to get parameters, any thoughts on the pattern we want to use to set them?

Reply:
One thought I had was to allow users to specify a property in the command and to handle it dynamically. E.g. node.name would get the name property, and if a parameter (like newValue) was provided to set the property

Reply:
For commands we should follow the API. Since setting attributes is not really translatable to an API, maybe we can just make it `set_<attribute name>` for them.